plotly/plotly/plotly.py

```diff
@@ -1308,7 +1308,19 @@ def response_handler(cls, response):
                     .format(url=get_config()['plotly_api_domain'])
                 )
             else:
-                raise requests_exception
+                try:
+                    parsed_response = response.json()
+                except:
+                    parsed_response = response.content
+
+                try:
+                    # try get a friendly error message
+                    errors = parsed_response.get('errors')
+                    message = errors[-1]['message']
+                except:
+                    raise requests_exception
+
+                raise exceptions.PlotlyError(message)
```

Comment:
Thanks for adding this to the central error handler, I actually forgot we even had it!